db/migrate/20220709110140_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :post_id
+      t.integer :author_id
```

Comment:
- [ ] Kindly reference these two columns to their primary keys in their respective tables